Sources/DangerShellExecutor/ShellExecutor.swift

```diff
@@ -35,24 +51,26 @@ public struct ShellExecutor: ShellExecuting {
 
     public func execute(_ command: String,
                         arguments: [String],
-                        environmentVariables: [String: String]) -> String {
-        let task = makeTask(for: command, with: arguments, environmentVariables: environmentVariables)
+                        environmentVariables: [String: String],
+                        outputFile: String?) -> String {
+        let task = makeTask(for: command, with: arguments, environmentVariables: environmentVariables, outputFile: outputFile)
 
         let pipe = Pipe()
         task.standardOutput = pipe
         task.launch()
         task.waitUntilExit()
 
         let data = pipe.fileHandleForReading.readDataToEndOfFile()
-        return String(data: data, encoding: .utf8)!.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
+        return String(data: data, encoding: .utf8)!.trimmingCharacters(in: .whitespacesAndNewlines)
     }
 
     // Similar to above, but can throw, and throws with most of
     // what you'd probably need in a scripting environment
     public func spawn(_ command: String,
                       arguments: [String],
-                      environmentVariables: [String: String]) throws -> String {
-        let task = makeTask(for: command, with: arguments, environmentVariables: environmentVariables)
+                      environmentVariables: [String: String],
+                      outputFile: String?) throws -> String {
+        let task = makeTask(for: command, with: arguments, environmentVariables: environmentVariables, outputFile: outputFile)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerShellExecutor/ShellExecutor.swift;
  Line: 73;
-->

- :warning: Line should be 120 characters or less: currently 126 characters (`line_length`)